plugin/src/main/java/com/denizenscript/denizen/objects/LocationTag.java

```diff
@@ -4318,6 +4318,38 @@ else if (material.hasModernData() && material.getModernData() instanceof org.buk
                 }
                 decoratedPot.update();
             });
+
+            // <--[tag]
+            // @attribute <LocationTag.buried_item>
+            // @returns ItemTag
+            // @mechanism LocationTag.buried_item
+            // @description
+            // Returns the item buried in a brushable block. Returns air if there is no item buried.
+            // -->
+            tagProcessor.registerTag(ItemTag.class, "buried_item", (attribute, object) -> {
+                if (!(object.getBlockStateForTag(attribute) instanceof BrushableBlock brushableBlock)) {
+                    return null;
+                }
+                return new ItemTag(brushableBlock.getItem());
+            });
+
+            // <--[mechanism]
+            // @object LocationTag
+            // @name buried_item
+            // @input ItemTag
+            // @description
+            // Sets the buried item in a brushable block.
```

Comment:
Considering the alternate naming of `suspicious block` used sometimes eg in minecraft wiki, it should probably be referenced somewhere in the meta (ie so someone who searches for 'suspicious block' will find this meta)